packages/dht/test/integration/StoreAndDelete.test.ts

```diff
@@ -42,6 +43,8 @@ describe('Storing data in DHT', () => {
         const successfulStorers = await storingNode.storeDataToDht(getDhtAddressFromRaw(entry.key), entry.data!)
         expect(successfulStorers.length).toBeGreaterThan(4)
         await storingNode.deleteDataFromDht(getDhtAddressFromRaw(entry.key), true)
+        // Wait for the delete operation to propagate
+        await wait(500)
```

Comment:
Would it make sense to use `waitForCondition`?

Reply:
Could do that, but it is not really necessary to do polling. Just need to wait a while for the delete operations to propagate